tests/test_arithmetics.rs

```diff
@@ -423,3 +437,57 @@ fn rich_comparisons_python_3_type_error() {
     py_expect_exception!(py, c2, "c2 >= 1", PyTypeError);
     py_expect_exception!(py, c2, "1 >= c2", PyTypeError);
 }
+
+#[test]
+fn rich_comparisons_python3_not_implemented() {
```

Comment:
Thank you, but isn't this test too long and complicated?
I recommend separating this into three test functions.

Reply:
I'll do it.